<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtMediumClass</class>
 <widget class="QWidget" name="QtMediumClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>530</width>
    <height>159</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtMedium</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>254</width>
     <height>43</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label1_2">
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>目标浓度(%)：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="concentration">
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;微软雅黑&quot;;</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>100</y>
     <width>222</width>
     <height>37</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label1_3">
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>酒精(ml)：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_water_3">
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;微软雅黑&quot;;
color: rgb(0,0,0);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="Do_acohol">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>100</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 14pt &quot;微软雅黑&quot;;</string>
   </property>
   <property name="text">
    <string>计算</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>211</width>
     <height>43</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label1">
      <property name="styleSheet">
       <string notr="true">font: 75 14pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="text">
       <string>水(ml)：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="water_vol">
      <property name="styleSheet">
       <string notr="true">font: 20pt &quot;微软雅黑&quot;;</string>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Normal</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
